2021q3/linuxulator.adoc

```diff
@@ -0,0 +1,24 @@
+=== Linux compatibility layer update
+
+Contact: Dmitry Chagin, <dchagin@FreeBSD.org> +
+Contact: Edward Tomasz Napierala, <trasz@FreeBSD.org>
+
+The goal of this project is to improve FreeBSD's ability to execute unmodified Linux binaries.
+Current support status of specific Linux applications is being tracked at the link:https://wiki.freebsd.org/LinuxApps[Linux app status Wiki page].
+
+The vDSO implementation was reworked.
+The futex support was overhauled to make use of `umtx` mechanism.
+It now supports priority-inheritance futexes, in addition to fixing several bugs.
+
+The `rt_sigsuspend(2)` and `sigaltstack(2)` syscalls are now supported on ARM64.
+
+The `faccessat2(2)`, `clone3(2)` system calls are now implemented.
+The `CLONE_CLEAR_RESETHAND` option is now supported.
+
+The `prctl(2)` syscall now supports `PR_SET_NO_NEW_PRIVS`.
+
+The `ptrace(2)` syscall now supports `PTRACE_GET_SYSCALL_INFO`, which is a prerequisite to support newer `strace(1)`.
+
+There is ongoing work to make Linuxulator on arm64 on par with the amd64 one; right now it's good enough for development work.
```

Comment:
Expand this maybe? Also clarify what kind of development work -- is this it's good enough for developing the Linuxulator, or it's good enough for running Linux/arm64 GUI IDEs to do development? Presumably it's that the compiler, rest of the tool chain, etc. work for compiling basic arm64 applications.

Reply:
Taste, but I would write "newer `strace(1)` versions".